index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="appBar">
+      <a href="#"><div class="logo-text">Jamila Moseka</div></a>
+      <a href="#"><img id="menu-icon" src="images/menu1.png" alt="logo" /></a>
+    </nav>
+
+    <section class="section-headline">
+      <h1 class="heading">Hey There. I'm Jamila</h1>
+      <p class="sub-heading">I am a Software Developer</p>
```

Comment:
- [ ] Please kindly make sure to align this section with `flexbox`.

Reply:
Fixed at index.html line 11-20